test/LocalizationWebsite/StartupResourcesInFolder.cs

```diff
@@ -26,7 +26,8 @@ public void ConfigureServices(IServiceCollection services)
             ILoggerFactory loggerFactory,
             IStringLocalizerFactory stringLocalizerFactory,
             IStringLocalizer<StartupResourcesInFolder> startupStringLocalizer,
-            IStringLocalizer<Customer> custromerStringLocalizer)
+            IStringLocalizer<Customer> custromerStringLocalizer,
+            IStringLocalizer<StartupCustomCulturePreserved> programLocalizer)
```

Comment:
Suggest adding a comment referencing #293 and spelling out the scenario this line checks.